File: CoteMultiLib/Model/Grade.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;

namespace CoreMultiLib.Model
{
    public class Grade
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public double? TheGrade { get; set; }

        public virtual Club Club { get; set; } = null!;
        public virtual Animal Animal { get; set; } = null!;

        public override string ToString()
        {
            return $"Grade : Id = {Id} Grade = {TheGrade}";
        }
    }
}

File: VBLib_Common/Model/Grade.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Model
    Public Class Grade
        <Key>
        <DatabaseGenerated(DatabaseGeneratedOption.Identity)>
        Public Property Id As Integer
        Public Property TheGrade As Double?
        Public Overridable Property Club As Club
        Public Overridable Property Animal As Animal

        Public Overrides Function ToString() As String
            Return $"Grade : Id = {Id} Grade = {TheGrade}"
        End Function
    End Class
End Namespace
